hms-app/src/actions/GetBackendData.tsx

```diff
@@ -0,0 +1,52 @@
+const api_id = '6cyvp5ukhc'
+
+export const getListOfHackathons = (endpoint: any) => {
+  return fetch(
+    `${'http://localhost:4566/restapis/'}${api_id}${'/local/_user_request_'}/${endpoint}`,
```

Comment:
so since you aren't executing any JS code or referencing variables you don't need the `${...}` around things like `http://local...` or `/loca/_user...`.

Same thing goes for the other places like that